### PR TITLE
Remove virtual client/connection creation

### PR DESCRIPTION
**Pull request by 7AC**
This is done by the relayer as part of https://github.com/polymerdao/polymerase/pull/819, we need that PR to merge and a tag for it. All that's left for the SDK to do is to detect the resulting client/connection IDs

### CODE REVIEW

File: src/lib/relayers/eth.ts [hunk 1]
```diff
@@ -134,89 +134,53 @@ export class EthRelayer {
...
-
-    for (let i = 0; i < 3; i++) {
-      try {
-        await this.createVirtualConnection(account.Address, signer, client)
+    if (!this.run.virtualClientId) throw new Error(`could not find altair virtual light client`)
+
+    const connectionsResponse = await ibcClient.connection.Connections(QueryIbcConnectionsRequest.fromPartial({}))
+    for (const connection of connectionsResponse.connections) {
+      // We look for the connection for the virtual end, which has the native light client for polymer
+      if (connection.clientId === 'polymer-0') {
```
Comment (Inkvi) [inline src/lib/relayers/eth.ts:171]: Is it an assumption that always works?

**Review by Inkvi — APPROVED** [review]

File: src/lib/relayers/eth.ts (hunk 1 above)
Comment (7AC) [reply src/lib/relayers/eth.ts:171]: yes https://github.com/polymerdao/ibcx-go/blob/57472d2eddca21943c766f2a3f93c6c976b0bdd7/modules/core/exported/client.go#L32

File: src/lib/relayers/eth.ts [hunk 2]
```diff
@@ -134,89 +134,53 @@ export class EthRelayer {
...
-      offlineSigner,
-      self.cosmos.client.signerOpts()
-    )
+    const client = await newTendermintClient(poly.Nodes[0].RpcHost)
+    const queryClient = QueryClient.withExtensions(client, setupPolyIbcExtension)
 
-    const clients = await queryClient.polyibc.ClientStates(
-      self.cosmos.client.polyibc.query.QueryClientStatesRequest.fromPartial({})
-    )
+    let clients = await queryClient.polyibc.ClientStates(QueryPolyIbcClientStatesRequest.fromPartial({}))
```
Comment (nicopernas) [inline src/lib/relayers/eth.ts:146]: nit: make `clients` const.

File: src/lib/relayers/eth.ts [hunk 3]
```diff
@@ -134,89 +134,53 @@ export class EthRelayer {
...
+    if (!this.run.nativeClientId) throw new Error(`could not find altair native light client`)
 
-    for (let i = 0; i < 3; i++) {
-      try {
-        await this.createVirtualLightClient(account.Address, signer, client)
-        break
-      } catch (e) {
-        log.warn(`could not create virtual light client: ${e}, retrying...`)
+    const ibcClient = QueryClient.withExtensions(client, setupIbcExtension)
+    clients = await ibcClient.client.ClientStates(QueryIbcClientStatesRequest.fromPartial({}))
```
Comment (nicopernas) [inline src/lib/relayers/eth.ts:159]: nit: and use a new variable here.

**Review by nicopernas — APPROVED** [review]

**Pull request merged**